### PR TITLE
[serde generate] add config option `custom_code`

### PR DESCRIPTION
**Pull request by matbd**
## Summary

Add option to inject custom code in various languages.

This is similar to comments, except inserted at the end of a class and without quotes.

## Test Plan

unit test

### CODE REVIEW

File: serde-generate/src/python3.rs [hunk 1]
```diff
@@ -252,7 +268,7 @@ import typing
         writeln!(self.out, "\nclass {}:", name)?;
         self.out.indent();
         self.output_comment(&name)?;
-        if self.generator.config.serialization {
+        if self.generator.config.serialization && !self.has_custom_code(&name) {
```
Comment (xli) [inline serde-generate/src/python3.rs:271]: why this guard? when has custom code, we lost serialization?

Comment (matbd) [reply serde-generate/src/python3.rs:271]: We output `pass` in the else branch.

Comment (xli) [reply serde-generate/src/python3.rs:271]: or, you think we should just pull out serialization into custom code?

Comment (matbd) [reply serde-generate/src/python3.rs:271]: Oh my bad it should be a OR.

**Review by xli — APPROVED** [review]

**Pull request merged**